static/templates/settings/deactivation_user_modal.hbs

```diff
@@ -1,11 +1,12 @@
 <div id="deactivation_user_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivation_user_modal_label" aria-hidden="true">
     <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="deactivation_user_modal_label">{{t "Deactivate" }} <span class="email"></span></h3>
+        <h3 id="deactivation_user_modal_label">{{t "Deactivate user" }} <span class="user_name"></span> <span class="email"></span></h3>
     </div>
     <div class="modal-body">
-        <p>{{#tr this}}By deactivating <strong><span class="user_name"></span></strong> &lt;<span class="email"></span>&gt;, they will be logged out immediately.{{/tr}}</p>
+        <p>{{#tr this}}Deactivating user <strong><span class="user_name"></span></strong> &lt;<span class="email"></span>&gt; will immediately log them out.{{/tr}}</p>
```

Comment:
![image](https://user-images.githubusercontent.com/59444243/114294001-a6523f00-9ab8-11eb-815a-03a69d4272c3.png)

Reply:
@alya Can you please verify this screenshot, Is it what we want exactly to show in the header?
(P.S: Puppeeter tests are failing for this file only, if this is final then I will work on that puppeeter test as it looks a bit complex thing to fix that puppeeter tests)

Reply:
I was thinking of having angle brackets around the email in the header, same as in the body. Thanks!